src/misclibs.F

```diff
@@ -3899,17 +3899,16 @@ subroutine calcmass(nstat,rstat,ruh,rvh,rmh,rho)
 
       tmass=0.0d0 
       !$omp parallel do default(shared) private(i,j,k)
-      !$acc parallel default(present) private(i,j,k) reduction(+:foot,tmass)
-      !$acc loop gang reduction(+:tmass)
+      !$acc parallel loop default(present) collapse(3) reduction(+:tmass)
       do k=1,nk
-        foot=0.0d0
-        !$acc loop vector collapse(2) reduction(+:foot)
+        !foot=0.0d0
+        !!$acc loop vector collapse(2) reduction(+:foot)
```

Comment:
remove these two lines

Reply:
@sjsprecious Thanks for catching these several cleanup items.